README.md

```diff
@@ -144,6 +144,12 @@ To run the phpunit tests:
 ```bash
 docker-compose exec web ./vendor/bin/phpunit
 ```
+
+### Generate changelog
+```bash
+docker-compose exec web ./vendor/bin/taskman github:changelog
```

Comment:
We cannot run docker inside the web Container :)
Maybe we should add a note about it ?